updates/28/updates.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "Monerium EURe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "A", 100, "0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", 18, "None"); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "EURe", "monerium-eur-money", "EURE", NULL, 1724565650, NULL);
```

Comment:
it's coming as "None" in all of them, probably we should have a test to check if any nullable field is "None"? Because we use a lot of Python code to create them
```suggestion
INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "Monerium EURe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "A", 100, "0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "EURe", "monerium-eur-money", "EURE", NULL, 1724565650, NULL);
```